lib/commons/color/color.js

```diff
@@ -83,6 +107,31 @@ function Color(red, green, blue, alpha) {
 		}
 	};
 
+	this.parseHexString = function(colorString) {
```

Comment:
NBD either way, but now that we can `import` things, I'm curious why you didn't grab something from npm rather than building this yourself.

Reply:
Security / package size. I know this does exactly what we need, and nothing else.

Reply:
Our bundler should be tree-shaking any unused dependency code out for us, and we would "bundle" it at build-time so it'd be easier to analyze the sources. I understand the security concern, but am just worried about adding more to the list of complex things we're responsible for maintaining. Not worth blocking this PR, but I thought once we'd migrated to modules, we'd stop "reinventing the wheel".